From bunk@kernel.org Wed Feb 13 13:30:55 2008
From: Adrian Bunk <bunk@kernel.org>
Date: Wed, 13 Feb 2008 23:30:12 +0200
Subject: PCI: #if 0 pci_assign_resource_fixed()
To: gregkh@suse.de
Cc: linux-pci@atrey.karlin.mff.cuni.cz
Message-ID: <20080213213012.GY3383@cs181133002.pp.htv.fi>
Content-Disposition: inline


An unused function that bloated the kernel only when CONFIG_EMBEDDED was
enabled...

Signed-off-by: Adrian Bunk <bunk@kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/pci/setup-res.c |    2 +-
 include/linux/pci.h     |    1 -
 2 files changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/pci/setup-res.c
+++ b/drivers/pci/setup-res.c
@@ -171,7 +171,7 @@ int pci_assign_resource(struct pci_dev *
 	return ret;
 }
 
-#ifdef CONFIG_EMBEDDED
+#if 0
 int pci_assign_resource_fixed(struct pci_dev *dev, int resno)
 {
 	struct pci_bus *bus = dev->bus;
--- a/include/linux/pci.h
+++ b/include/linux/pci.h
@@ -602,7 +602,6 @@ int pcie_get_readrq(struct pci_dev *dev)
 int pcie_set_readrq(struct pci_dev *dev, int rq);
 void pci_update_resource(struct pci_dev *dev, struct resource *res, int resno);
 int __must_check pci_assign_resource(struct pci_dev *dev, int i);
-int __must_check pci_assign_resource_fixed(struct pci_dev *dev, int i);
 int pci_select_bars(struct pci_dev *dev, unsigned long flags);
 
 /* ROM control related routines */
